### PR TITLE
Admin screen

### PR DESCRIPTION
**Pull request by arinarudevich**
Admin home screen

### CODE REVIEW

File: src/components/header/header.css [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+	align-items: center;
+}
+
+.header-content__switch-mode-button{
+  text-align: center;
+	border:  2px solid #008AFF;
+	border-radius: 2px;
+	color: white;
+	cursor: pointer;
+	font: 400 24px 'Roboto', sans-serif;
```
Comment (armenbadalyan) [inline src/components/header/header.css]: Login pull request #6 is adding Tajawal font. Please decide with the team members which one you will use.

File: src/components/header/header.hbs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                <span class="header-content__balance">{{balance}}<span class="header-content__currency">P</span></span>
+            </div>
+            <i class="history-ico material-icons">&#xE889;</i>
+            {{else}}
+            <span class="header-content__logout-label">Выйти</span>
+            {{/unless}}
+            <i class="exit-ico material-icons md-25">&#xE879;</i>
+        </div>
+
+        {{!link "Подробнее..." poem.url}}
```
Comment (armenbadalyan) [inline src/components/header/header.hbs]: ?

File: src/components/header/header.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import Handlebars from 'handlebars';
+import template from './header.hbs';
+
+// Handlebars.registerHelper("isManager", function(context, options) {
+//   if(context.userRole){
+//     return options.fn(this);
+//   }
+// });
+
+Handlebars.registerHelper('link', function(text, options) {
```
Comment (armenbadalyan) [inline src/components/header/header.js]: Do we need this helper?

File: src/screens/admin/admin.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+export default class AdminHomeScreen {
+    constructor() {
+    }
+
+    render(screenTarget, props) {
+        const adminContent = AdminContent();
+        screenTarget.innerHTML = adminContent;
+        const header = new Header();
+        header.render(screenTarget, props);
+        return true;
```
Comment (Stkulibaba) [inline src/screens/admin/admin.js]: Better return screenTarget instead of true.

**Pull request merged**